### DISCUSSION TITLE
Make it easier to find actions on mobile

### DISCUSSION THREAD

**Discussion by dginovker** [community/community#5775] [Mobile] [answered]
One of my projects has CICD that builds an APK each commit. Finding the artifact on the Android build is insanely difficult. Please make it easier!

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note and suggestion @dginovker — I agree, navigating into Actions and finding the logs + artifacts is really hard and tedious right now. We're not working on Actions right now, but I'll add your feedback to our tracking issues for when we do get started.
